src/Core/Translation/Builder/Map/Message.php

```diff
@@ -76,6 +81,18 @@ public function setUserTranslation(string $userTranslation): self
         return $this;
     }
 
+    public function getMetadata(): array
```

Comment:
You should update https://github.com/PrestaShop/PrestaShop/blob/b81b9dda9688598ff6199a96615ea5552fc2294e/tests/Unit/Core/Translation/Builder/Map/MessageTest.php

Reply:
In Message class it's just a getter and a setter, no need for unit test. I tested it throught TranslationCatalogueBuilder in which we set and retrieve the metadata

Reply:
I agree with @sowbiba , just like [Robert C Martin](https://blog.cleancoder.com/uncle-bob/2013/03/06/ThePragmaticsOfTDD.html) said

> I don’t write tests for getters and setters. Those getters and setters will be indirectly tested by the tests of the other methods; so there’s no point in testing them directly.